doc/digraph.xml

```diff
@@ -1083,3 +1083,145 @@ gap> JohnsonDigraph(1,0);
   </Description>
 </ManSection>
 <#/GAPDoc>
+
+<#GAPDoc Label="DigraphNumber">
+<ManSection>
+  <Oper Name="DigraphNumber" Arg="nr[, v]"/>
+  <Returns>A digraph.</Returns>
+  <Description>
+    This operation defines a bijection between the positive integers and the set
+    of all digraphs without multiple edges.  If <A>nr</A> is a positive integer,
+    then this operation returns the unique digraph which corresponds to that
+    number.  This result will be consistent across sessions and systems.  The
+    digraphs will ordered by increasing number of vertices. <P/>
+
+    If the optional argument <A>v</A> is specified and is a non-negative
+    integer, then this operation will use a different mapping which only assigns
+    numbers to digraphs with <A>v</A> vertices (with no multiple edges).  If
```

Comment:
I don't like the part after "then this operation..." could this be rewritten to simply define the mapping rather than say a "different mapping which only assigns..."